package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "remark-embed-images",
   "version": "3.0.0",
-  "description": "remark plugin to embed images with data URIs, inlining base64-encoded sources",
+  "description": "remark plugin to embed local images as data URIs, inlining files as encoded (base64) values",
```

similar with ordering
```suggestion
  "description": "remark plugin to embed local images as data URIs, inlining files as (base64) encoded values",
```